### PR TITLE
feat: Sign in screen

### PR DESCRIPTION
**Pull request by cecipirotto**
# :star: Feature
Sign in screen
---

#### iOS

<p align="center">

https://github.com/xmartlabs/supabase-ws-flutter/assets/31603856/f39b0701-c04f-44a7-90a5-470d0b77b9be

</p>
---

#### :pushpin: Notes:

Pending: 
- Add icons to textfields, header and footer
- Add navigation to sign up screen

### CODE REVIEW

File: supabase-ws-flutter-mobile/lib/ui/home/home_screen.dart [hunk 1]
```diff
@@ -47,8 +47,9 @@ class _WelcomeContentScreenState extends State<_WelcomeContentScreen> {
               IconButton(
                 icon: const Icon(Icons.more_vert),
                 color: context.theme.colors.textColor.shade100,
-                //TODO: add logic
-                onPressed: () => {},
+                onPressed: () => {
+                  context.read<HomeCubit>().logOut(),
+                },
```
Comment (mirland) [inline supabase-ws-flutter-mobile/lib/ui/home/home_screen.dart:52]: use just the lamda

File: supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart [hunk 2]
```diff
@@ -23,26 +26,49 @@ class _SignInContentScreen extends StatelessWidget {
...
+                children: [
+                  const SizedBox(
+                    height: 180,
+                  ),
+                  _SignInForm(),
+                  if (context.read<SignInCubit>().state.error.isNotEmpty)
+                    Text(
+                      context.localizations
+                          .error(context.read<SignInCubit>().state.error),
+                    ),
```
Comment (mirland) [inline supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart:43]: how are you showing the errors? I'd show just an alert to be consistent

File: supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart [hunk 3]
```diff
@@ -23,26 +26,49 @@ class _SignInContentScreen extends StatelessWidget {
...
-                  child: Text(context.localizations.sign_in),
-                ),
+          body: SafeArea(
+            child: Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Column(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  const SizedBox(
+                    height: 180,
```
Comment (mirland) [inline supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart:36]: should't you center or something like that?

File: supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart [hunk 4]
```diff
@@ -23,26 +26,49 @@ class _SignInContentScreen extends StatelessWidget {
...
+                  _SignInForm(),
+                  if (context.read<SignInCubit>().state.error.isNotEmpty)
+                    Text(
+                      context.localizations
+                          .error(context.read<SignInCubit>().state.error),
+                    ),
+                  Row(
+                    children: [
+                      Expanded(
+                        child: AppPrimaryButton(
```
Comment (mirland) [inline supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart:47]: you are not using the factories

File: supabase-ws-flutter-mobile/.fvm/flutter_sdk [hunk 5]
```diff
@@ -0,0 +1 @@
+/Users/ceciliapirotto/fvm/versions/3.13.4
```
Comment (mirland) [inline supabase-ws-flutter-mobile/.fvm/flutter_sdk:1]: you cannot track it

**Review by mirland — CHANGES_REQUESTED** [review]
Check new files

File: supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart (hunk 2 above)
Comment (cecipirotto) [reply supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart]: we are going to update in sign in and sign up later

**Review by mirland — APPROVED** [review]

**Pull request merged**